packages/fluentui/react-northstar/src/components/Portal/Portal.tsx

```diff
@@ -99,7 +99,7 @@ export const Portal: React.FC<PortalProps> & FluentComponentStaticProps<PortalPr
 
   const renderPortal = (): JSX.Element | undefined => {
     const contentToRender = childrenExist(children) ? children : content;
-    const focusTrapZoneProps = (_.keys(trapFocus).length && trapFocus) || {};
```

Comment:
this was causing `error TS2698: Spread types may only be created from object types` on ` {trapFocus ? <FocusTrapZone {...focusTrapZoneProps}>{contentToRender}</FocusTrapZone> : contentToRender}`